### ISSUE TITLE
Not sure if low_hi is working as intended

### ISSUE THREAD

**Issue by jsdillon** [HERA-Team/hera_cal#362]
I'm not sure that low_hi is doing what we want here. In particular, it may return a list of baselines, some of which need to be complex conjugated in order for all of them to be redundant. Perhaps what we really want is for low_hi to make sure that the first baseline i,j in a redundant group is i < j... and perhaps that the group is sorted by i.

This affects

https://github.com/HERA-Team/hera_cal/blob/f113198d28d2031c81ce6dd2b7f37bb15ef8ecf6/hera_cal/abscal_funcs.py#L725

and also

https://github.com/HERA-Team/hera_cal/blob/f113198d28d2031c81ce6dd2b7f37bb15ef8ecf6/hera_cal/abscal_funcs.py#L1572

@nkern what do you think?

**Comment by nkern** [comment]
agreed, I fixed this in a commit in the `abscal_lstbin_h2c_prep` branch

**Comment by jsdillon** [comment]
So, I actually came up with a different fix: https://github.com/HERA-Team/hera_cal/blob/3b9b8543139f3cfdab318c44d34e514eff70701f/hera_cal/redcal.py#L98

I think we can unify this and get rid of the low_hi option. 

`return [sorted(reds[delta]) if sorted(reds[delta])[0][0] == np.min(reds[delta])
                else sorted([reverse_bl(bl) for bl in reds[delta]]) for delta in orderedDeltas]`
